pkg/loop/internal/core/services/capability/standard_capability.go

```diff
@@ -0,0 +1,223 @@
+package capability
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/grpc"
+	"google.golang.org/protobuf/types/known/emptypb"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	capabilitiespb "github.com/smartcontractkit/chainlink-common/pkg/capabilities/pb"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/core/services/errorlog"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/core/services/keyvalue"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/core/services/pipeline"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/core/services/telemetry"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/goplugin"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/net"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb"
+	relayersetpb "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb/relayerset"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/relayerset"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/core"
+)
+
+type StandardCapability interface {
+	services.Service
+	capabilities.BaseCapability
+	Initialise(ctx context.Context, config string, telemetryService core.TelemetryService, store core.KeyValueStore,
```

Comment:
is config a string because it's up to the implementation to interpret?

Reply:
If it is capability instance config, then the answer would be "yes" as on-chain it's stored as bytes. I'm not sure why it isn't `[]byte` here. A more precise name for `config` would be 👌

Reply:
Correct, its up to the implementation to interpret, hence string, possible it would be json or similar specific to that standard capability type in practice.  

wrt to if it should be []byte - possibly.  One concern is that none of the existing job specs use []byte as an attribute type as I would guess using []byte would be problematic for the operator UI.